
32   2.22507385850720240&-308 0.00000000000000000&+000  up
32   2.22507385850720280&-308   put o get o put, i=   4

32   2.22507385850720560&-308 0.00000000000000000&+000  up
32   2.22507385850720520&-308   put o get o put, i=   8

32   2.22507385850720720&-308 0.00000000000000000&+000  up
32   2.22507385850720680&-308   put o get o put, i=  11
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
16   2.22507385850720680&-308 0.00000000000000000&+000  up
16   2.22507385850720640&-308   put o get o put, i=  11

32   2.22507385850720880&-308 0.00000000000000000&+000  up
32   2.22507385850720920&-308   put o get o put, i=  17

16   2.22507385850721320&-308 0.00000000000000000&+000  up
16   2.22507385850721280&-308   put o get o put, i=  24

32   2.22507385850721320&-308 0.00000000000000000&+000  up
32   2.22507385850721360&-308   put o get o put, i=  26

32   2.22507385850721520&-308 0.00000000000000000&+000  up
32   2.22507385850721560&-308   put o get o put, i=  30

32   2.22507385850721760&-308 0.00000000000000000&+000  up
32   2.22507385850721720&-308   put o get o put, i=  32

16   2.22507385850721840&-308 0.00000000000000000&+000  up
16   2.22507385850721880&-308   put o get o put, i=  37

32   2.22507385850721960&-308 0.00000000000000000&+000  up
32   2.22507385850722000&-308   put o get o put, i=  39

16   2.22507385850722160&-308 0.00000000000000000&+000  up
16   2.22507385850722120&-308   put o get o put, i=  41

32   2.22507385850722280&-308 0.00000000000000000&+000  up
32   2.22507385850722240&-308   put o get o put, i=  43

32   2.22507385850722360&-308 0.00000000000000000&+000  up
32   2.22507385850722400&-308   put o get o put, i=  47

32   2.22507385850722840&-308 0.00000000000000000&+000  up
32   2.22507385850722800&-308   put o get o put, i=  54

16   2.22507385850722920&-308 0.00000000000000000&+000  up
16   2.22507385850722960&-308   put o get o put, i=  59

16   2.22507385850722960&-308 0.00000000000000000&+000  up
16   2.22507385850723000&-308   put o get o put, i=  60
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850723000&-308 0.00000000000000000&+000  up
32   2.22507385850723040&-308   put o get o put, i=  60

16   2.22507385850723240&-308 0.00000000000000000&+000  up
16   2.22507385850723200&-308   put o get o put, i=  63

16   2.22507385850723400&-308 0.00000000000000000&+000  up
16   2.22507385850723360&-308   put o get o put, i=  66

32   2.22507385850723800&-308 0.00000000000000000&+000  up
32   2.22507385850723840&-308   put o get o put, i=  76

16   2.22507385850724000&-308 0.00000000000000000&+000  up
16   2.22507385850723960&-308   put o get o put, i=  78

16   2.22507385850724000&-308 0.00000000000000000&+000  up
16   2.22507385850724040&-308   put o get o put, i=  81

16   2.22507385850724120&-308 0.00000000000000000&+000  up
16   2.22507385850724160&-308   put o get o put, i=  83

16   2.22507385850724440&-308 0.00000000000000000&+000  up
16   2.22507385850724400&-308   put o get o put, i=  87

32   2.22507385850724560&-308 0.00000000000000000&+000  up
32   2.22507385850724520&-308   put o get o put, i=  89

16   2.22507385850724600&-308 0.00000000000000000&+000  up
16   2.22507385850724640&-308   put o get o put, i=  93
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850724640&-308 0.00000000000000000&+000  up
32   2.22507385850724680&-308   put o get o put, i=  93

16   2.22507385850725080&-308 0.00000000000000000&+000  up
16   2.22507385850725040&-308   put o get o put, i= 100

32   2.22507385850725080&-308 0.00000000000000000&+000  up
32   2.22507385850725120&-308   put o get o put, i= 102

16   2.22507385850725240&-308 0.00000000000000000&+000  up
16   2.22507385850725280&-308   put o get o put, i= 106
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850725280&-308 0.00000000000000000&+000  up
32   2.22507385850725320&-308   put o get o put, i= 106

16   2.22507385850725520&-308 0.00000000000000000&+000  up
16   2.22507385850725480&-308   put o get o put, i= 109

16   2.22507385850725680&-308 0.00000000000000000&+000  up
16   2.22507385850725720&-308   put o get o put, i= 115
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850725720&-308 0.00000000000000000&+000  up
32   2.22507385850725760&-308   put o get o put, i= 115

32   2.22507385850726480&-308 0.00000000000000000&+000  up
32   2.22507385850726440&-308   put o get o put, i= 128

16   2.22507385850726720&-308 0.00000000000000000&+000  up
16   2.22507385850726680&-308   put o get o put, i= 133

16   2.22507385850727320&-308 0.00000000000000000&+000  up
16   2.22507385850727360&-308   put o get o put, i= 148
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850727360&-308 0.00000000000000000&+000  up
32   2.22507385850727400&-308   put o get o put, i= 148

16   2.22507385850727760&-308 0.00000000000000000&+000  up
16   2.22507385850727720&-308   put o get o put, i= 154

16   2.22507385850727800&-308 0.00000000000000000&+000  up
16   2.22507385850727760&-308   put o get o put, i= 155

16   2.22507385850728080&-308 0.00000000000000000&+000  up
16   2.22507385850728040&-308   put o get o put, i= 161
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850728120&-308 0.00000000000000000&+000  up
32   2.22507385850728080&-308   put o get o put, i= 161

32   2.22507385850728320&-308 0.00000000000000000&+000  up
32   2.22507385850728280&-308   put o get o put, i= 165

32   2.22507385850728760&-308 0.00000000000000000&+000  up
32   2.22507385850728720&-308   put o get o put, i= 174

16   2.22507385850728960&-308 0.00000000000000000&+000  up
16   2.22507385850729000&-308   put o get o put, i= 181

32   2.22507385850729400&-308 0.00000000000000000&+000  up
32   2.22507385850729360&-308   put o get o put, i= 187

16   2.22507385850729440&-308 0.00000000000000000&+000  up
16   2.22507385850729480&-308   put o get o put, i= 191
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850729480&-308 0.00000000000000000&+000  up
32   2.22507385850729520&-308   put o get o put, i= 191

16   2.22507385850729600&-308 0.00000000000000000&+000  up
16   2.22507385850729640&-308   put o get o put, i= 194

16   2.22507385850730040&-308 0.00000000000000000&+000  up
16   2.22507385850730080&-308   put o get o put, i= 203

16   2.22507385850730360&-308 0.00000000000000000&+000  up
16   2.22507385850730320&-308   put o get o put, i= 207
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850730400&-308 0.00000000000000000&+000  up
32   2.22507385850730360&-308   put o get o put, i= 207

32   2.22507385850730560&-308 0.00000000000000000&+000  up
32   2.22507385850730600&-308   put o get o put, i= 213

16   2.22507385850731000&-308 0.00000000000000000&+000  up
16   2.22507385850730960&-308   put o get o put, i= 220
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850731040&-308 0.00000000000000000&+000  up
32   2.22507385850731000&-308   put o get o put, i= 220

16   2.22507385850731640&-308 0.00000000000000000&+000  up
16   2.22507385850731600&-308   put o get o put, i= 233

32   2.22507385850731640&-308 0.00000000000000000&+000  up
32   2.22507385850731680&-308   put o get o put, i= 235

32   2.22507385850732080&-308 0.00000000000000000&+000  up
32   2.22507385850732040&-308   put o get o put, i= 241

32   2.22507385850732200&-308 0.00000000000000000&+000  up
32   2.22507385850732240&-308   put o get o put, i= 246

16   2.22507385850732640&-308 0.00000000000000000&+000  up
16   2.22507385850732600&-308   put o get o put, i= 253
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850732680&-308 0.00000000000000000&+000  up
32   2.22507385850732640&-308   put o get o put, i= 253

32   2.22507385850732680&-308 0.00000000000000000&+000  up
32   2.22507385850732720&-308   put o get o put, i= 256

32   2.22507385850732840&-308 0.00000000000000000&+000  up
32   2.22507385850732880&-308   put o get o put, i= 259

32   2.22507385850733120&-308 0.00000000000000000&+000  up
32   2.22507385850733080&-308   put o get o put, i= 262
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
16   2.22507385850733080&-308 0.00000000000000000&+000  up
16   2.22507385850733040&-308   put o get o put, i= 262

32   2.22507385850733160&-308 0.00000000000000000&+000  up
32   2.22507385850733120&-308   put o get o put, i= 263

16   2.22507385850733240&-308 0.00000000000000000&+000  up
16   2.22507385850733280&-308   put o get o put, i= 268
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850733280&-308 0.00000000000000000&+000  up
32   2.22507385850733320&-308   put o get o put, i= 268

16   2.22507385850733800&-308 0.00000000000000000&+000  up
16   2.22507385850733840&-308   put o get o put, i= 279

32   2.22507385850733920&-308 0.00000000000000000&+000  up
32   2.22507385850733960&-308   put o get o put, i= 281

16   2.22507385850734120&-308 0.00000000000000000&+000  up
16   2.22507385850734080&-308   put o get o put, i= 283

32   2.22507385850734240&-308 0.00000000000000000&+000  up
32   2.22507385850734200&-308   put o get o put, i= 285

32   2.22507385850734320&-308 0.00000000000000000&+000  up
32   2.22507385850734360&-308   put o get o put, i= 289

16   2.22507385850734760&-308 0.00000000000000000&+000  up
16   2.22507385850734720&-308   put o get o put, i= 296

32   2.22507385850734880&-308 0.00000000000000000&+000  up
32   2.22507385850734840&-308   put o get o put, i= 298

16   2.22507385850734880&-308 0.00000000000000000&+000  up
16   2.22507385850734920&-308   put o get o put, i= 301

16   2.22507385850734920&-308 0.00000000000000000&+000  up
16   2.22507385850734960&-308   put o get o put, i= 302
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850734960&-308 0.00000000000000000&+000  up
32   2.22507385850735000&-308   put o get o put, i= 302

16   2.22507385850735200&-308 0.00000000000000000&+000  up
16   2.22507385850735160&-308   put o get o put, i= 305

16   2.22507385850735360&-308 0.00000000000000000&+000  up
16   2.22507385850735320&-308   put o get o put, i= 308

16   2.22507385850735400&-308 0.00000000000000000&+000  up
16   2.22507385850735360&-308   put o get o put, i= 309

32   2.22507385850735400&-308 0.00000000000000000&+000  up
32   2.22507385850735440&-308   put o get o put, i= 311

16   2.22507385850735840&-308 0.00000000000000000&+000  up
16   2.22507385850735800&-308   put o get o put, i= 318

16   2.22507385850735960&-308 0.00000000000000000&+000  up
16   2.22507385850736000&-308   put o get o put, i= 323

16   2.22507385850736400&-308 0.00000000000000000&+000  up
16   2.22507385850736360&-308   put o get o put, i= 329

16   2.22507385850736480&-308 0.00000000000000000&+000  up
16   2.22507385850736440&-308   put o get o put, i= 331

16   2.22507385850737040&-308 0.00000000000000000&+000  up
16   2.22507385850737000&-308   put o get o put, i= 342

32   2.22507385850737040&-308 0.00000000000000000&+000  up
32   2.22507385850737080&-308   put o get o put, i= 344

16   2.22507385850737480&-308 0.00000000000000000&+000  up
16   2.22507385850737440&-308   put o get o put, i= 351

16   2.22507385850737640&-308 0.00000000000000000&+000  up
16   2.22507385850737680&-308   put o get o put, i= 357
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850737680&-308 0.00000000000000000&+000  up
32   2.22507385850737720&-308   put o get o put, i= 357

32   2.22507385850738000&-308 0.00000000000000000&+000  up
32   2.22507385850737960&-308   put o get o put, i= 361

32   2.22507385850738440&-308 0.00000000000000000&+000  up
32   2.22507385850738400&-308   put o get o put, i= 370

32   2.22507385850738640&-308 0.00000000000000000&+000  up
32   2.22507385850738600&-308   put o get o put, i= 374

16   2.22507385850738640&-308 0.00000000000000000&+000  up
16   2.22507385850738680&-308   put o get o put, i= 377

32   2.22507385850739080&-308 0.00000000000000000&+000  up
32   2.22507385850739040&-308   put o get o put, i= 383

16   2.22507385850739280&-308 0.00000000000000000&+000  up
16   2.22507385850739320&-308   put o get o put, i= 390

16   2.22507385850739720&-308 0.00000000000000000&+000  up
16   2.22507385850739760&-308   put o get o put, i= 399

16   2.22507385850739920&-308 0.00000000000000000&+000  up
16   2.22507385850739960&-308   put o get o put, i= 403

32   2.22507385850740280&-308 0.00000000000000000&+000  up
32   2.22507385850740240&-308   put o get o put, i= 407

32   2.22507385850740280&-308 0.00000000000000000&+000  up
32   2.22507385850740320&-308   put o get o put, i= 410

16   2.22507385850740680&-308 0.00000000000000000&+000  up
16   2.22507385850740640&-308   put o get o put, i= 416
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850740720&-308 0.00000000000000000&+000  up
32   2.22507385850740680&-308   put o get o put, i= 416

32   2.22507385850741360&-308 0.00000000000000000&+000  up
32   2.22507385850741320&-308   put o get o put, i= 429

16   2.22507385850741560&-308 0.00000000000000000&+000  up
16   2.22507385850741600&-308   put o get o put, i= 436

16   2.22507385850742200&-308 0.00000000000000000&+000  up
16   2.22507385850742240&-308   put o get o put, i= 449

32   2.22507385850742520&-308 0.00000000000000000&+000  up
32   2.22507385850742560&-308   put o get o put, i= 455

32   2.22507385850742840&-308 0.00000000000000000&+000  up
32   2.22507385850742800&-308   put o get o put, i= 459

16   2.22507385850742960&-308 0.00000000000000000&+000  up
16   2.22507385850742920&-308   put o get o put, i= 462
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850743000&-308 0.00000000000000000&+000  up
32   2.22507385850742960&-308   put o get o put, i= 462

32   2.22507385850743600&-308 0.00000000000000000&+000  up
32   2.22507385850743640&-308   put o get o put, i= 477

32   2.22507385850743920&-308 0.00000000000000000&+000  up
32   2.22507385850743880&-308   put o get o put, i= 481

32   2.22507385850744040&-308 0.00000000000000000&+000  up
32   2.22507385850744000&-308   put o get o put, i= 483

16   2.22507385850744240&-308 0.00000000000000000&+000  up
16   2.22507385850744200&-308   put o get o put, i= 488

32   2.22507385850744640&-308 0.00000000000000000&+000  up
32   2.22507385850744680&-308   put o get o put, i= 498

32   2.22507385850744800&-308 0.00000000000000000&+000  up
32   2.22507385850744840&-308   put o get o put, i= 501

16   2.22507385850745040&-308 0.00000000000000000&+000  up
16   2.22507385850745000&-308   put o get o put, i= 504
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850745080&-308 0.00000000000000000&+000  up
32   2.22507385850745040&-308   put o get o put, i= 504

32   2.22507385850745120&-308 0.00000000000000000&+000  up
32   2.22507385850745080&-308   put o get o put, i= 505

16   2.22507385850745200&-308 0.00000000000000000&+000  up
16   2.22507385850745240&-308   put o get o put, i= 510

16   2.22507385850745680&-308 0.00000000000000000&+000  up
16   2.22507385850745640&-308   put o get o put, i= 517

16   2.22507385850745760&-308 0.00000000000000000&+000  up
16   2.22507385850745800&-308   put o get o put, i= 521

32   2.22507385850746200&-308 0.00000000000000000&+000  up
32   2.22507385850746160&-308   put o get o put, i= 527

16   2.22507385850746240&-308 0.00000000000000000&+000  up
16   2.22507385850746280&-308   put o get o put, i= 531
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850746280&-308 0.00000000000000000&+000  up
32   2.22507385850746320&-308   put o get o put, i= 531

16   2.22507385850746280&-308 0.00000000000000000&+000  up
16   2.22507385850746320&-308   put o get o put, i= 532

16   2.22507385850746720&-308 0.00000000000000000&+000  up
16   2.22507385850746680&-308   put o get o put, i= 538

32   2.22507385850746840&-308 0.00000000000000000&+000  up
32   2.22507385850746800&-308   put o get o put, i= 540

32   2.22507385850746920&-308 0.00000000000000000&+000  up
32   2.22507385850746960&-308   put o get o put, i= 544

16   2.22507385850747320&-308 0.00000000000000000&+000  up
16   2.22507385850747280&-308   put o get o put, i= 550

16   2.22507385850747320&-308 0.00000000000000000&+000  up
16   2.22507385850747360&-308   put o get o put, i= 553
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850747360&-308 0.00000000000000000&+000  up
32   2.22507385850747400&-308   put o get o put, i= 553

16   2.22507385850747480&-308 0.00000000000000000&+000  up
16   2.22507385850747520&-308   put o get o put, i= 556

16   2.22507385850747800&-308 0.00000000000000000&+000  up
16   2.22507385850747760&-308   put o get o put, i= 560

32   2.22507385850748120&-308 0.00000000000000000&+000  up
32   2.22507385850748080&-308   put o get o put, i= 566

16   2.22507385850748560&-308 0.00000000000000000&+000  up
16   2.22507385850748600&-308   put o get o put, i= 578

16   2.22507385850748960&-308 0.00000000000000000&+000  up
16   2.22507385850749000&-308   put o get o put, i= 586
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850748960&-308 0.00000000000000000&+000  up
32   2.22507385850749040&-308   put o get o put, i= 586

16   2.22507385850749600&-308 0.00000000000000000&+000  up
16   2.22507385850749640&-308   put o get o put, i= 599
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850749640&-308 0.00000000000000000&+000  up
32   2.22507385850749680&-308   put o get o put, i= 599

32   2.22507385850749960&-308 0.00000000000000000&+000  up
32   2.22507385850749920&-308   put o get o put, i= 603

16   2.22507385850750040&-308 0.00000000000000000&+000  up
16   2.22507385850750080&-308   put o get o put, i= 608

32   2.22507385850750400&-308 0.00000000000000000&+000  up
32   2.22507385850750360&-308   put o get o put, i= 612

32   2.22507385850750600&-308 0.00000000000000000&+000  up
32   2.22507385850750560&-308   put o get o put, i= 616

16   2.22507385850750600&-308 0.00000000000000000&+000  up
16   2.22507385850750640&-308   put o get o put, i= 619

32   2.22507385850751040&-308 0.00000000000000000&+000  up
32   2.22507385850751000&-308   put o get o put, i= 625

32   2.22507385850751120&-308 0.00000000000000000&+000  up
32   2.22507385850751160&-308   put o get o put, i= 629

16   2.22507385850751240&-308 0.00000000000000000&+000  up
16   2.22507385850751280&-308   put o get o put, i= 632

32   2.22507385850751680&-308 0.00000000000000000&+000  up
32   2.22507385850751640&-308   put o get o put, i= 638

16   2.22507385850751680&-308 0.00000000000000000&+000  up
16   2.22507385850751720&-308   put o get o put, i= 641

16   2.22507385850751880&-308 0.00000000000000000&+000  up
16   2.22507385850751920&-308   put o get o put, i= 645

16   2.22507385850752200&-308 0.00000000000000000&+000  up
16   2.22507385850752160&-308   put o get o put, i= 649

16   2.22507385850752320&-308 0.00000000000000000&+000  up
16   2.22507385850752360&-308   put o get o put, i= 654

16   2.22507385850752640&-308 0.00000000000000000&+000  up
16   2.22507385850752600&-308   put o get o put, i= 658
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850752680&-308 0.00000000000000000&+000  up
32   2.22507385850752640&-308   put o get o put, i= 658

16   2.22507385850753280&-308 0.00000000000000000&+000  up
16   2.22507385850753240&-308   put o get o put, i= 671

32   2.22507385850753280&-308 0.00000000000000000&+000  up
32   2.22507385850753320&-308   put o get o put, i= 673

32   2.22507385850753720&-308 0.00000000000000000&+000  up
32   2.22507385850753680&-308   put o get o put, i= 679

32   2.22507385850753840&-308 0.00000000000000000&+000  up
32   2.22507385850753880&-308   put o get o put, i= 684

16   2.22507385850754160&-308 0.00000000000000000&+000  up
16   2.22507385850754200&-308   put o get o put, i= 691

32   2.22507385850754360&-308 0.00000000000000000&+000  up
32   2.22507385850754320&-308   put o get o put, i= 692

32   2.22507385850754480&-308 0.00000000000000000&+000  up
32   2.22507385850754520&-308   put o get o put, i= 697

32   2.22507385850754800&-308 0.00000000000000000&+000  up
32   2.22507385850754760&-308   put o get o put, i= 701

16   2.22507385850754920&-308 0.00000000000000000&+000  up
16   2.22507385850754880&-308   put o get o put, i= 704

32   2.22507385850754920&-308 0.00000000000000000&+000  up
32   2.22507385850754960&-308   put o get o put, i= 706

16   2.22507385850754920&-308 0.00000000000000000&+000  up
16   2.22507385850754960&-308   put o get o put, i= 707
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850754960&-308 0.00000000000000000&+000  up
32   2.22507385850755000&-308   put o get o put, i= 707

32   2.22507385850755440&-308 0.00000000000000000&+000  up
32   2.22507385850755400&-308   put o get o put, i= 714

32   2.22507385850755560&-308 0.00000000000000000&+000  up
32   2.22507385850755600&-308   put o get o put, i= 719

32   2.22507385850756000&-308 0.00000000000000000&+000  up
32   2.22507385850755960&-308   put o get o put, i= 725

16   2.22507385850756080&-308 0.00000000000000000&+000  up
16   2.22507385850756120&-308   put o get o put, i= 730

16   2.22507385850756400&-308 0.00000000000000000&+000  up
16   2.22507385850756360&-308   put o get o put, i= 734

32   2.22507385850756520&-308 0.00000000000000000&+000  up
32   2.22507385850756480&-308   put o get o put, i= 736

32   2.22507385850756600&-308 0.00000000000000000&+000  up
32   2.22507385850756640&-308   put o get o put, i= 740

32   2.22507385850757080&-308 0.00000000000000000&+000  up
32   2.22507385850757040&-308   put o get o put, i= 747

16   2.22507385850757160&-308 0.00000000000000000&+000  up
16   2.22507385850757200&-308   put o get o put, i= 752

16   2.22507385850757200&-308 0.00000000000000000&+000  up
16   2.22507385850757240&-308   put o get o put, i= 753
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850757240&-308 0.00000000000000000&+000  up
32   2.22507385850757280&-308   put o get o put, i= 753

16   2.22507385850757480&-308 0.00000000000000000&+000  up
16   2.22507385850757440&-308   put o get o put, i= 756

16   2.22507385850757640&-308 0.00000000000000000&+000  up
16   2.22507385850757600&-308   put o get o put, i= 759

16   2.22507385850758120&-308 0.00000000000000000&+000  up
16   2.22507385850758080&-308   put o get o put, i= 769
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850758160&-308 0.00000000000000000&+000  up
32   2.22507385850758120&-308   put o get o put, i= 769

16   2.22507385850758240&-308 0.00000000000000000&+000  up
16   2.22507385850758280&-308   put o get o put, i= 774

16   2.22507385850758680&-308 0.00000000000000000&+000  up
16   2.22507385850758640&-308   put o get o put, i= 780

32   2.22507385850758680&-308 0.00000000000000000&+000  up
32   2.22507385850758720&-308   put o get o put, i= 782

16   2.22507385850759280&-308 0.00000000000000000&+000  up
16   2.22507385850759240&-308   put o get o put, i= 792

16   2.22507385850759280&-308 0.00000000000000000&+000  up
16   2.22507385850759320&-308   put o get o put, i= 795
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850759320&-308 0.00000000000000000&+000  up
32   2.22507385850759360&-308   put o get o put, i= 795

16   2.22507385850759480&-308 0.00000000000000000&+000  up
16   2.22507385850759520&-308   put o get o put, i= 799
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850759520&-308 0.00000000000000000&+000  up
32   2.22507385850759560&-308   put o get o put, i= 799

16   2.22507385850759760&-308 0.00000000000000000&+000  up
16   2.22507385850759720&-308   put o get o put, i= 802

32   2.22507385850760080&-308 0.00000000000000000&+000  up
32   2.22507385850760040&-308   put o get o put, i= 808

32   2.22507385850760280&-308 0.00000000000000000&+000  up
32   2.22507385850760240&-308   put o get o put, i= 812

32   2.22507385850760720&-308 0.00000000000000000&+000  up
32   2.22507385850760680&-308   put o get o put, i= 821

16   2.22507385850760920&-308 0.00000000000000000&+000  up
16   2.22507385850760960&-308   put o get o put, i= 828

16   2.22507385850761560&-308 0.00000000000000000&+000  up
16   2.22507385850761600&-308   put o get o put, i= 841

16   2.22507385850762000&-308 0.00000000000000000&+000  up
16   2.22507385850762040&-308   put o get o put, i= 850

16   2.22507385850762320&-308 0.00000000000000000&+000  up
16   2.22507385850762280&-308   put o get o put, i= 854
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850762360&-308 0.00000000000000000&+000  up
32   2.22507385850762320&-308   put o get o put, i= 854

32   2.22507385850762520&-308 0.00000000000000000&+000  up
32   2.22507385850762560&-308   put o get o put, i= 860

32   2.22507385850763000&-308 0.00000000000000000&+000  up
32   2.22507385850762960&-308   put o get o put, i= 867

16   2.22507385850763200&-308 0.00000000000000000&+000  up
16   2.22507385850763240&-308   put o get o put, i= 874

32   2.22507385850763600&-308 0.00000000000000000&+000  up
32   2.22507385850763640&-308   put o get o put, i= 882

32   2.22507385850763840&-308 0.00000000000000000&+000  up
32   2.22507385850763800&-308   put o get o put, i= 884

16   2.22507385850763840&-308 0.00000000000000000&+000  up
16   2.22507385850763880&-308   put o get o put, i= 887

16   2.22507385850764160&-308 0.00000000000000000&+000  up
16   2.22507385850764120&-308   put o get o put, i= 891

16   2.22507385850764280&-308 0.00000000000000000&+000  up
16   2.22507385850764320&-308   put o get o put, i= 896

16   2.22507385850764600&-308 0.00000000000000000&+000  up
16   2.22507385850764560&-308   put o get o put, i= 900
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850764640&-308 0.00000000000000000&+000  up
32   2.22507385850764600&-308   put o get o put, i= 900

32   2.22507385850764800&-308 0.00000000000000000&+000  up
32   2.22507385850764840&-308   put o get o put, i= 906

16   2.22507385850765240&-308 0.00000000000000000&+000  up
16   2.22507385850765200&-308   put o get o put, i= 913

32   2.22507385850765240&-308 0.00000000000000000&+000  up
32   2.22507385850765280&-308   put o get o put, i= 915

16   2.22507385850765760&-308 0.00000000000000000&+000  up
16   2.22507385850765800&-308   put o get o put, i= 926

32   2.22507385850765880&-308 0.00000000000000000&+000  up
32   2.22507385850765920&-308   put o get o put, i= 928

32   2.22507385850766120&-308 0.00000000000000000&+000  up
32   2.22507385850766080&-308   put o get o put, i= 930

32   2.22507385850766320&-308 0.00000000000000000&+000  up
32   2.22507385850766280&-308   put o get o put, i= 934

16   2.22507385850766520&-308 0.00000000000000000&+000  up
16   2.22507385850766480&-308   put o get o put, i= 939

32   2.22507385850766760&-308 0.00000000000000000&+000  up
32   2.22507385850766720&-308   put o get o put, i= 943

32   2.22507385850766920&-308 0.00000000000000000&+000  up
32   2.22507385850766960&-308   put o get o put, i= 949

32   2.22507385850767080&-308 0.00000000000000000&+000  up
32   2.22507385850767120&-308   put o get o put, i= 952

16   2.22507385850767320&-308 0.00000000000000000&+000  up
16   2.22507385850767280&-308   put o get o put, i= 955
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850767360&-308 0.00000000000000000&+000  up
32   2.22507385850767320&-308   put o get o put, i= 955

32   2.22507385850767400&-308 0.00000000000000000&+000  up
32   2.22507385850767360&-308   put o get o put, i= 956

16   2.22507385850767400&-308 0.00000000000000000&+000  up
16   2.22507385850767440&-308   put o get o put, i= 959

16   2.22507385850767920&-308 0.00000000000000000&+000  up
16   2.22507385850767880&-308   put o get o put, i= 967
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850767960&-308 0.00000000000000000&+000  up
32   2.22507385850767920&-308   put o get o put, i= 967

16   2.22507385850768040&-308 0.00000000000000000&+000  up
16   2.22507385850768080&-308   put o get o put, i= 972

32   2.22507385850768160&-308 0.00000000000000000&+000  up
32   2.22507385850768200&-308   put o get o put, i= 974

16   2.22507385850768360&-308 0.00000000000000000&+000  up
16   2.22507385850768320&-308   put o get o put, i= 976

32   2.22507385850768480&-308 0.00000000000000000&+000  up
32   2.22507385850768440&-308   put o get o put, i= 978

16   2.22507385850768520&-308 0.00000000000000000&+000  up
16   2.22507385850768560&-308   put o get o put, i= 982
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850768560&-308 0.00000000000000000&+000  up
32   2.22507385850768600&-308   put o get o put, i= 982

16   2.22507385850769000&-308 0.00000000000000000&+000  up
16   2.22507385850768960&-308   put o get o put, i= 989
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
32   2.22507385850769040&-308 0.00000000000000000&+000  up
32   2.22507385850769000&-308   put o get o put, i= 989

32   2.22507385850769200&-308 0.00000000000000000&+000  up
32   2.22507385850769240&-308   put o get o put, i= 995

16   2.22507385850769440&-308 0.00000000000000000&+000  up
16   2.22507385850769400&-308   put o get o put, i= 998

*** SIGNIFICANCE from  0.00000000000000000&+000  up    is 17

*** SIGNIFICANCE from  1.79769313486231560&+308  down  is 17

*** SIGNIFICANCE from  3.45000000000000000&+000  up    is 17

*** SIGNIFICANCE from  3.45000000000000000&+000  down  is 17

*** SIGNIFICANCE from  4.49999900000000000&+000  up    is 17

*** SIGNIFICANCE from  4.49999900000000000&+000  down  is 17

*** SIGNIFICANCE from  1.00000000000000000&+001  up    is 17

*** SIGNIFICANCE from  1.00000000000000000&+001  down  is 17

*** SIGNIFICANCE from  1.00000000000000000&+031  up    is 17

*** SIGNIFICANCE from  1.00000000000000000&+041  up    is 17

..END Simula a.s test 100 (no significant deviations found)


*** SIGNIFICANCE from  0.00000000000000000&+000  up    is 16

*** SIGNIFICANCE from  1.79769313486231520&+308  down  is 17

*** SIGNIFICANCE from  3.45000000000000000&+000  up    is 17

*** SIGNIFICANCE from  3.45000000000000000&+000  down  is 17

*** SIGNIFICANCE from  4.49999900000000000&+000  up    is 17

*** SIGNIFICANCE from  4.49999900000000000&+000  down  is 17

*** SIGNIFICANCE from  1.00000000000000000&+001  up    is 17

*** SIGNIFICANCE from  1.00000000000000000&+001  down  is 17

*** SIGNIFICANCE from  1.00000000000000000&+031  up    is 17

*** SIGNIFICANCE from  1.00000000000000000&+041  up    is 17


 Number of significant digits varies between 16 and 17

